src/components/mdpTimePicker/mdpTimePicker.js

```diff
@@ -250,12 +250,12 @@ module.provider("$mdpTimePicker", function() {
                 locals: {
                     time: time,
                     autoSwitch: options.autoSwitch,
-                    ampm: options.ampm
+                    ampm: options.ampm == null ? $mdpLocale.time.ampm : options.ampm
```

Comment:
I think we might go better with `angular.isDefined(options.ampm) ? options.ampm : $mdpLocale.time.ampm` here